apps/simplex-chat/Simplex/Chat.hs

```diff
@@ -160,6 +170,68 @@ chatSubscriber = do
             unsetActive $ ActiveC c
           _ -> pure ()
       _ -> pure ()
+  where
+    saveContact userId activeConn connInfo = do
```

Comment:
would this not shadow `connInfo` from line 153?

Reply:
No, Haskell only has function parameters in scope, not the variables that were defined in the function body.